magefile.go

```diff
@@ -335,7 +340,6 @@ func EnsureGinkgo() {
 func setupTestNamespace() {
 	SetupNamespace(testNamespace)
 	EnsureTestSecret()
-	SetupTestCredentialSet()
```

Comment:
Hey @bdegeeter , do you mind to explain why this setup is necessary here?

Reply:
@VinozzZ What unwanted behavior are you seeing because of where this is being called. Is the problem its called too soon and needs to be moved closer to the test itself? Or is it just not needed to be explicitly called before the tests are run?

Reply:
This is applying data before the new porter agent is installed in the cluster. So the database ends up with an older schema.
I just checked that this should not be necessary since the data is created again in each integration test https://github.com/getporter/kubernetes-plugins/blob/3855eddabf6dbc5935758c49cc0a1ccf9d8ad2d8/tests/integration/operator/ginkgo/installation_test.go#L42